src/plugins/vis_type_timeseries/public/application/components/_vis_with_splits.scss

```
@@ -23,3 +23,12 @@
     min-height: $tvbSplitBlockVisMinSize / 1.25;
   }
 }
+
+.tvbSplitVis__split__one {
+  display: flex;
+  flex: 1 0 25%;
+}
+
+.tvbSplitVis__one {
+  flex: 1;
+}
```

Then you can just modify what you need to with one selector.

```suggestion
// When there is only one visualization, expand the full height and remove all minimum sizes
.tvbSplitVis--one {
  flex: 1;
  
  .tvbSplitVis__split {
    min-width: 0;
    
    > .tvbVis {
      min-height: 0;
    }
  }
}
```

---

done